stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -139,43 +132,46 @@ class FPDF:
         font_cache_dir: bool = ...,
     ) -> None: ...
     @property
-    def unifontsubset(self): ...
+    def unifontsubset(self) -> bool: ...
     @property
-    def epw(self): ...
+    def epw(self) -> float: ...
     @property
-    def eph(self): ...
+    def eph(self) -> float: ...
     def set_margin(self, margin: float) -> None: ...
     def set_margins(self, left: float, top: float, right: float = ...) -> None: ...
+    l_margin: float
     def set_left_margin(self, margin: float) -> None: ...
+    t_margin: float
     def set_top_margin(self, margin: float) -> None: ...
+    r_margin: float
     def set_right_margin(self, margin: float) -> None: ...
-    auto_page_break: Any
-    b_margin: Any
-    page_break_trigger: Any
+    auto_page_break: bool
+    b_margin: float
+    page_break_trigger: float
     def set_auto_page_break(self, auto: bool, margin: float = ...) -> None: ...
-    zoom_mode: Any
-    layout_mode: Any
-    def set_display_mode(self, zoom, layout: str = ...) -> None: ...
-    def set_compression(self, compress) -> None: ...
-    title: Any
+    zoom_mode: str | float
```

Comment:
Could be more precise here with Literal types: https://github.com/PyFPDF/fpdf2/blob/48f837963b7ae1956a0e5b23815063d23c1964f5/fpdf/fpdf.py#L587

But that might also lead to false positives.

Reply:
```suggestion
    zoom_mode: Literal["fullpage", "fullwidth", "real", "default"] | float
    page_layout: PageLayout | None
```